admob/integration_test/src/integration_test.cc

```diff
@@ -729,10 +731,23 @@ TEST_F(FirebaseAdMobTest, TestInterstitialAdLoadAndShow) {
   EXPECT_EQ(full_screen_content_listener.num_on_ad_showed_full_screen_content_,
             1);
   EXPECT_EQ(full_screen_content_listener.num_on_ad_impression_, 1);
+  EXPECT_EQ(full_screen_content_listener
+                .num_on_ad_failed_to_show_full_screen_content_,
+            0);
   EXPECT_EQ(paid_event_listener.num_on_paid_event_, 1);
   EXPECT_EQ(
       full_screen_content_listener.num_on_ad_dismissed_full_screen_content_, 1);
 
+  // Show the Ad again
+  LogDebug("Attempting to show ad again, checking for correct error result.");
+  WaitForCompletion(interstitial->Show(), "Show");
+  app_framework::ProcessEvents(2000);
+  EXPECT_EQ(full_screen_content_listener.failure_codes_.size(), 1);
```

Comment:
You can use gMock matchers for this...

EXPECT_THAT(full_screen_content_listener.failure_codes_, ElementsAre(firebase::admob::kAdMobErrorAdAlreadyUsed));

Reply:
(also I'd make an accessor rather than using failure_codes_ directly...)

Reply:
Done.